src/components/scenes/Loans/ManageCollateralScene.tsx

```diff
@@ -43,22 +43,42 @@ import { FormScene } from '../FormScene'
 
 type ActionOpType = 'loan-borrow' | 'loan-deposit' | 'loan-repay' | 'loan-withdraw'
 
+// User input display strings
+const ACTIONOP_TYPE_STRING_MAP = {
+  'loan-borrow': {
+    headerText: s.strings.loan_borrow_more,
+    amountCard: s.strings.loan_fragment_loan,
+    srcDestCard: s.strings.loan_fund_destination
+  },
+  'loan-deposit': {
+    headerText: s.strings.loan_add_collateral,
+    amountCard: s.strings.loan_fragment_deposit,
+    srcDestCard: s.strings.loan_fund_source
+  },
+  'loan-repay': {
+    headerText: s.strings.loan_make_payment,
+    amountCard: s.strings.loan_fragment_repay,
+    srcDestCard: s.strings.loan_fund_source
+  },
+  'loan-withdraw': {
+    headerText: s.strings.loan_withdraw_collateral,
+    amountCard: s.strings.loan_fragment_withdraw,
+    srcDestCard: s.strings.loan_fund_destination
+  }
+}
+
 type Props<T extends keyof ParamList> = {
   actionOpType: ActionOpType
-  amountChange?: 'increase' | 'decrease'
   loanAccount: LoanAccount
 
-  showAprChange?: boolean
-
-  headerText: string
   navigation: NavigationProp<T>
 }
 
 export const ManageCollateralScene = <T extends keyof ParamList>(props: Props<T>) => {
   // -----------------------------------------------------------------------------
   // #region Constants
   // -----------------------------------------------------------------------------
-  const { actionOpType, amountChange = 'increase', loanAccount, showAprChange = false, headerText, navigation } = props
+  const { actionOpType, loanAccount, navigation } = props
```

Comment:
Bravo! I love the reduction in complexity made to this component. It's a very useful abstraction now.